ingestion/src/metadata/ingestion/source/metadata/atlas.py

```diff
@@ -266,6 +266,13 @@ def _parse_table_entity(self, name, entity):  # pylint: disable=too-many-locals
                         entity=Table, fqn=table_fqn
                     )
 
+                    tag_fqn = fqn.build(
+                        self.metadata,
+                        entity_type=Tag,
+                        tag_category_name=ATLAS_TAG_CATEGORY,
+                        tag_name=ATLAS_TABLE_TAG,
+                    )
+
```

Comment:
add a top condition checking if table object is there..
if not, we do nothing, show warning

if we have table_object,
then we check for description,
complete that,

create tag_fqn, and then patch_tags

Reply:
this way, we make less calls to the ES, we check for description, if not there, we create tag fqn (call to es) and then patch_tags

Reply:
Oh ok, let me do that.